webview.h

```diff
@@ -1771,6 +1826,19 @@ class webview2_com_handler
     }
     return S_OK;
   }
+  HRESULT STDMETHODCALLTYPE Invoke(
+      ICoreWebView2 *sender, ICoreWebView2NavigationCompletedEventArgs *args) {
+    PWSTR uri = nullptr;
+    auto hr = sender->get_Source(&uri);
+    if (SUCCEEDED(hr)) {
+      auto curi =
```

Comment:
What does the `c` in `curi` mean?

Reply:
as in `C string` type, but can rename.

Reply:
Okay. Since `to_bytes()` should return `std::string` here, I suggest renaming it to avoid the confusion.